### PR TITLE
records: remove `indexer_class` from configuration

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: backend/inspirehep/records/config.py [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "default_endpoint_prefix": True,
     # XXX decide about the links
     "links_factory_imp": "inspirehep.records.links:inspire_detail_links_factory",
-    "indexer_class": RecordIndexer,
+    "indexer_class": None,
```
Comment (michamos) [inline backend/inspirehep/records/config.py:74]: should we add a test for this (e.g. mocking the indexer task to do nothing, then checking that the record is not in ES)?

Comment (drjova) [reply backend/inspirehep/records/config.py:74]: IMHO it's invenio configuraiton, If we start testing this then we should add tests for many others

Comment (michamos) [reply backend/inspirehep/records/config.py:74]: ok then, except for the now (hopefully) unused import on L18 :)

**Review by michamos — APPROVED** [review]

File: backend/inspirehep/records/config.py (hunk 1 above)
Comment (drjova) [reply backend/inspirehep/records/config.py:74]: 🤦

**Pull request merged**